marray/__init__.py

```diff
@@ -79,20 +79,26 @@ def __setitem__(self, key, other):
             self.mask[key] = getattr(other, 'mask', False)
             return self.data.__setitem__(key, getattr(other, 'data', other))
 
+        def _data_mask_string(self, fun):
+            data_str = fun(self.data) + ", "
+            mask_str = fun(self.mask)
+            if len(data_str) + len(mask_str) <= 66:
+                join_char = ""
+                components = ["MaskedArray(", data_str, mask_str, ")"]
```

Comment:
instead of using `str.join` I would have gone with an f-string because that makes it easier to understand the end result. Something like
```suggestion
                result = f"MaskedArray({data_str}, {mask_str})"
```

Reply:
I suppose that looks better here. I am not as convinced about the multiline version, though.